main.py

```diff
@@ -10,6 +9,7 @@
     display = '-' * len(key)
     if game_start == "play":
         i = 0
+        used_letters = []
```

Comment:
Instead of having a global variable which you need to clear, it's better to make the variable local to the scope where you use it. This gives you automatic clearance when it leaves scope.